index.html

```diff
@@ -0,0 +1,362 @@
+    <div id="main-posts">
+        <div class="posts-items">
+            <h1>Grabbing Visual attention with the visual cortex</h1>
+            <span>by Susan Weinschenk</span>
+            <p>Given the way our brains work, there are things you can do that will grab your user’s visual attention.
+                In this
+                article, Susan Weinschenk explains how the visual cortex of our brains plays a vital role in
+                controlling our
+                behavior.</p>
+        </div>
+        <div class="posts-items">
+            <h1>HTML5 Input Types: Where Are They Now?</h1>
+            <span>by Drew McLellan</span>
+            <p>HTML5 introduced thirteen new types of form input, adding significantly to the number of different
+                fields web
+                designers and developers could add to our forms. But what is the state of those field types in 2019?
+                Let’s find out.
+            </p>
+        </div>
+        <div class="posts-items">
+            <h1>Web Standards: The What, The Why, And The How</h1>
+            <span>by Amy Dickens</span>
+            <p>Web Standards, and the documentation to support them, provide huge insight into ‘the why’ and ‘the what’
+                of the
+                world wide web. In this article, we take a look at the history of Web Standards, how to use them in
+                your work and
+                ways you can get …</p>
+        </div>
+        <div class="posts-items">
+            <div>ADVERT</div>
+        </div>
+        <div class="posts-items-big">
+            <h1>1,358 smashing <br> members</h1>
+            <p>A friendly community that keeps this website alive. With books, webinars, discounts and networking -
+                just 1
+                coffee a month. Join the family.</p>
+            <div>Explore feature &bkarow;</div>
+            <p>Already have an account?<a href="#">Sign in!</a></p>
+        </div>
+        <div class="posts-items">
+            <h1>When And how to use CSS Multi-Column layout</h1>
+            <span>by Rachel Andrew</span>
+            <p>The Multi-column Layout spec is often overlooked as we use Grid and Flexbox. In this article Rachel
+                Andrew
+                explains why it is different to other layout methods, and shows some useful patterns and sites which
+                showcase it
+                well.</p>
+        </div>
+        <div class="posts-items">
+            <h1>Designing a font based on old handwriting</h1>
+            <span>by Carolyn Porter</span>
+            <p>Are you considering transforming old cursive handwriting into a modern digital font? In this article,
+                Carolyn Porter gives advice on what to keep in mind when designing and releasing a new font.</p>
+        </div>
+    </div>
+  <!-- main post ends -->
+    <div class="clear"></div>
+  <!-- upcoming event start -->
+    <div class="event-box">
+        <br><br><br><br>
```

Comment:
This is impractical. If we wanted to modify height, how would that look on this system? Adding `br` tags until we get it right? **Use CSS for height modification and not `br`**